src/github/auto-approve.ts

```diff
@@ -0,0 +1,54 @@
+import { Component } from '../component';
+import { Project } from '../project';
+
+/**
+ * Options for 'AutoApprove'
+ */
+export interface AutoApproveOptions {
+
+  /**
+   * PR's assigned with this label will be auto-approved.
+   *
+   * @default pr/auto-approve
```

Comment:
Defaults should be JSON encoded (and also, please change to `auto-approve`)

```suggestion
   * @default "auto-approve"
```